cetmix_tower_server/models/cx_tower_variable_value.py

```diff
@@ -87,6 +89,15 @@ class TowerVariableValue(models.Model):
         ),
     ]
 
+    @api.constrains("required", "value_char")
+    def _check_required_values(self):
```

Comment:
According to the task required  variable values are used only in the wizard that creates new server from template.
Why do we need this constraint here?

Reply:
You’re right, this constraint is redundant since required variable validation is only used in the server creation wizard. I’ll remove it to avoid unnecessary checks at the model level.